# Rainwater Collection System Design

In this recipe, we present design strategies for sizing a Rainwater Collection System for capturing and storing rainfall for domestic uses, e.g. graywater or irrigation use.  Rainfall collection systems are diverse and generally require a number of considerations in the design phase.  In the recipe below, we assume a design consisting of the following elements:

* A rooftop collection system
* A water storage tank
* Appropriate plumbing, filtration, and pumping systems


## Design Strategy

Our design strategy presented below focuses on the following strategy.

1. Determine the rooftop area available for rainwater capture.  This imposes a constraint on the availability of water for storage
2. Estimate the water demand that the rainwater collection system can offset
3. Estimate cost associated with construction, maintenance and disposal of the system, and associated savings from covering this demand with the collection system
4. Determine the most cost effective size configuration.


## Design Parameters

In the example below, we assume we are designing a system for Portland, Oregon, for a small apartment building with a flat gravel roof with a surface area of 1800 m2. The design is intend to offset water used for 1) graywater (toilets) and 2) landscape irrigation.  The apartment building houses 60 residents. The area of landscaping to be irrigated by the system is 0.4 ha (4000 m2).



## Relevant Standards

Standards for design of rainwater collection systems are currently lacking in the US, but have been developed for other countries, particularly the Portugal (Ref 1), UK (Ref 2), and Germany (Ref 3).  Additionally, a number of articles in peer-reviewed journals address design of these systems.  See the references provided at the bottom of this page for additional information. 

## Simplified Approach

In this approach, we follow Ref 3 below to size the system. This approach has the advantage of simplicity, but does not consider seasonal variability that can be important if the seasonality of supply and demand are out of phase. This approach involves the following steps:

1. Estimate rainwater yield based on roof area, type, and annual precipitation
2. Estimate demand based on use cases
3. Economically optimize the size of the system


### Step 1. Estimate annual rainwater yield

We estimate the annual water yield (water available for collection for this system as follows:

$\displaystyle \large E_R = A_A \times e \times h_n \times \eta $

where: 
* $E_R$ is the rainwater yield in liters per year (l/yr)
* $A_A$ is the collecting area in square meters (m2)
* $e$ is the yield fraction (decimal percent), see table 1 below.
* $h_N$ is the quantity of precipitation in liters per square meter (liters/m2, or mm)
* $\eta$ is the hydraulic filter efficiency (0.90 is a typical value)

Values of $e$ can be estimated from the table below:

<table style='fontsize:large'><tr><th>Composition</th><th>Yield Coefficient (percent)</th></tr>
    <tr><td>Slanted hard roof<sup>a</sup></td><td>0.8</td></tr>
    <tr><td>Flat roof, without gravel</td><td>0.8</td></tr>
    <tr><td>Flat roof, with gravel</td><td>0.6</td></tr>
    <tr><td>Green roof, intensive</td><td>0.3</td></tr>
    <tr><td>Green roof, extensive</td><td>0.5</td></tr>
    <tr><td>Paved surface/compound paved surface</td><td>0.5</td></tr>
    <tr><td>Asphalt covering</td><td>0.8</td></tr>
    <tr><td colspan=2><sup>a</sup>Deviations depending on the absorbency and roughness</td></tr>
</table>

For Portland, Oregon, we can get average annual precipitation from a number of sources. From the NOAA website (https://www.wrh.noaa.gov/pqr/pdxclimate/pg133.pdf), we see that the average annual precipitation ($h_N$) is 36.03 inches (915 mm).  Our roof area available for collection is 1800 m2.  From the table above, we choose an $e$ value of 0.6.  We can calculate the annual water yields as follows:  

In [1]:
annualPrecip = 915     # mm
collectionArea = 1800  # m2
yieldFraction = 0.6
filterEff = 0.9
annRainwaterYield =  collectionArea * yieldFraction * annualPrecip * filterEff   # liters/year
print("Average Annual Rainwater Yield for Portland Oregon is {:.0f} m3/year".format(annRainwaterYield/1000))

Average Annual Rainwater Yield for Portland Oregon is 889 m3/year


### Step 2: Estimate demand based on use cases

For this system, we assume demand for water for this system is based on 1) graywater (toilet) use, and 2) landscape irrigation. For graywater use, Ref 3 provides the following per-capita estimates:

| Use    | Daily Per-capita Consumption Rate | Specific Annual Requirements |
|--------|:---------------------------------:|:----------------------------:|
| Household Toilets | 24 liter/day | |
| Office Toilets    | 12 liters/day | |
| School Toilets    | 6 liters/day | |
| Garden Watering per m2 garden area | | 60 liter per m2|

Total annual water demand is the sum of "process water" (toilet) demand and landscape irrigation demand.

**Process water demand** is estimated using the following relationship:

$\displaystyle BW_p = P_d \times n \times 365 / 1000 $

where: 
* $BW_p$ is the annual process water demand (m3)
* $P_d$ is the daily per-capita requirements (L/day, from table above)

**Landscape irrigation demand** is estimated as follows:

$\displaystyle BW_i = A_i \times\ BS_i / 1000 $ 

where:
* $BW_i$ is the annual process water demand (m3)
* $A_i$ is the area to be irrigated (m2)
* $BS_i$ is the annuual per-area irrigation demand (liters/m2, from table above)

For our system, **total annual demand** for both process water and irrigation can be estimated as follows:

In [2]:
nResidents = 60
irrigArea = 400               # m2
perCapitaProcessDemand = 24   # liters/person/day
perAreaIrrigDemand = 60       # liters/m2.year
annProcessDemand = perCapitaProcessDemand * nResidents * 365 / 1000  # m3/year
annIrrigDemand = perAreaIrrigDemand * irrigArea / 1000  # m3/year
annTotalDemand = annProcessDemand + annIrrigDemand

print( "Annual Process Water Demand: {:.0f} m3/year".format(annProcessDemand))
print( "Annual Irrigation Water Demand: {:.0f} m3/year".format(annIrrigDemand))
print( "Annual Total Water Demand: {:.0f} m3/year".format(annTotalDemand))

Annual Process Water Demand: 526 m3/year
Annual Irrigation Water Demand: 24 m3/year
Annual Total Water Demand: 550 m3/year


### Step 3.  Estimate maximum required storage volume

We can see from the results above that annual available water is ~889 m<sup>3</sup>, and the annual demand is ~550 m<sup>3</sup>, so in principle we should be able to meet 100 percent of the demand. To estimate the  maximum storage volume requirement, we need to determine how long we can expect between rainfall events that replenish our storage, and demand during these "dry" periods, in other words, how much storage is needed to meet demand during these periods.  Once we have this period, we can size our storage take using the following:

$\displaystyle V_n = min( BW_t, E_R )/365 * P_{dry} $

where:
* $V_n$ is the maximum required storage volume (m3)
* $BW_t$ is the total annual water demand (m3/year)
* $E_R$ is the annual rainfall yield (m3/year)
* $P_{dry}$ = is the number of days between replenishing rainfall events (days)

Portland, Oregon's typical climate pattern is dry summers (when irrigation demand is high), and wet winters.  We will assume that during summer, we need to anticipate 2 months of storage between significant rain events, and therefore we estimate $P-{dry}$ as 2 months (60 days).  Note in the code below, we reference variables defined previously (above) in this notebook.

In [3]:
import numpy as np

pDry = 60  # days
storageVol = np.minimum(annTotalDemand, annRainwaterYield) * pDry / 365.0

print( "Required Storage Volume: {:.0f} m3".format(storageVol))

Required Storage Volume: 90 m3


### Step 4. Economic Analysis

Most designs require economic consideration, and we do so below for this problem.  We will generate a "Net Present Value" analysis and determine the payback period for our design.  We then look for a storage volume that is economically optimized, and repeat the analysis.

For these analyses, we assume the following:

* Cost of the **collection system** is \\$0.9/m2 for installation, \\$0.1/m2 annual maintenance, and \\$0.5/m2 for disposal at the end of life; installation costs include pipes, filters, etc.
* Cost of the **storage system** is \\$25/m3 for installation, \\$0.02/m3 annual maintenance, and \\$0.05/m3 for disposal at the end of life; installation costs include pipes, filters, etc.
* The value of water not purchased as a result of this system is \\$10 per 1000 ft3 (1 m3=35.3147 ft3) averaged over the lifetime of the project
* The design lifetime of the system is 20 years
* The discount rate for the investment is 5 percent/year

Our NPV analysis, with estimated payback period, follows:

In [16]:
import ecoe.econ as ee

discountRate = 0.05
installCost = 1.9*irrigArea + 30*storageVol     # $
maintCost = 0.1*irrigArea + 0.02*storageVol   # $/year
disposeCost = 0.05*irrigArea + 0.1*storageVol    # $
annSavings = annTotalDemand * 10 * 35.3147 / 1000 # $/year
lifetime = 20  # years

df = ee.GetNPVTable(discountRate,installCost,maintCost,disposeCost,annSavings,lifetime)




display(df)

Year Period Cost Period Revenue Net Cash Flow Cumulative Cash Flow  \
0     1     3470.36         194.09      -3276.27             -3276.27   
1     2     41.8069         194.09       152.283             -3123.98   
2     3     41.8069         194.09       152.283              -2971.7   
3     4     41.8069         194.09       152.283             -2819.42   
4     5     41.8069         194.09       152.283             -2667.14   
5     6     41.8069         194.09       152.283             -2514.85   
6     7     41.8069         194.09       152.283             -2362.57   
7     8     41.8069         194.09       152.283             -2210.29   
8     9     41.8069         194.09       152.283             -2058.01   
9    10     41.8069         194.09       152.283             -1905.72   
10   11     41.8069         194.09       152.283             -1753.44   
11   12     41.8069         194.09       152.283             -1601.16   
12   13     41.8069         194.09       152.283             -1448.87   
13   14     41.8069         194.09       152.283             -1296.59   
14   15     41.8069         194.09       152.283             -1144.31   
15   16     41.8069         194.09       152.283             -992.026   
16   17     41.8069         194.09       152.283             -839.744   
17   18     41.8069         194.09       152.283             -687.461   
18   19     41.8069         194.09       152.283             -535.178   
19   20     70.8414         194.09       123.248              -411.93   

   Period NPV Cumulative NPV  
0    -3120.25       -3120.25  
1     138.125       -2982.13  
2     131.548       -2850.58  
3     125.283        -2725.3  
4     119.317       -2605.98  
5     113.636       -2492.34  
6     108.224       -2384.12  
7     103.071       -2281.05  
8     98.1628       -2182.89  
9     93.4884        -2089.4  
10    89.0365       -2000.36  
11    84.7967       -1915.57  
12    80.7588       -1834.81  
13    76.9131       -1757.89  
14    73.2506       -1684.64  
15    69.7625       -1614.88  
16    66.4404       -1548.44  
17    63.2766       -1485.16  
18    60.2634        -1424.9  
19    46.4509       -1378.45



## References

1. ANQIP. (2009). ETA 0701 - Sistemas de Abastecimento de Águas Pluviais em Edifícios (SAAP) (in portuguese): Portuguese National Association for Quality in Building Services - ANQIP, Version 7.

2. BSI. (2009). BS 8515:2009 Rainwater harvesting systems - Code of practice. UK: British Standard Institution

3. Translation of DIN 1989-1:2001-10. Rainwater harvesting systems - Part 1: Planning, installation, operation and maintenance. Fachvereinigung Betriebs - und Regenwassernutzung e.V. Darmstadt, Germany. 34 pp.